### PR TITLE
add: Content for IBM MQ sink connector

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Describe your changes

Content for how to create an IBM MQ sink connector in Aiven for Apache Kafka/Aiven for Apache Kafka Connent. 

Ref: https://aiven.atlassian.net/browse/EC-419

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

**Review by MichaelTansiniAiven — APPROVED** [review]
it looks good to me.

File: docs/products/kafka/kafka-connect/howto/ibm-mq-sink-connector.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+
+The IBM MQ sink connector allows you to route messages from Apache Kafka® topics to IBM MQ queues.
+
+## Prerequisites {#connect_ibm_mq_sink_prereq}
+
+- An [Aiven for Apache Kafka service](enable-connect) with Apache Kafka Connect enabled,
```
Comment (ArthurFlag) [inline docs/products/kafka/kafka-connect/howto/ibm-mq-sink-connector.md:12]: can you use the full path?

File: docs/products/kafka/kafka-connect/howto/ibm-mq-sink-connector.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+1. On the **IBM MQ Sink** connector page, go to the **Common** tab.
+1. Locate the **Connector configuration** text box and click <ConsoleLabel name="edit"/>.
+1. Paste the configuration from your `ibm_mq_sink_connector.json` file into the text box.
+1. Click **Create connector**.
+1. Verify the connector status on the <ConsoleLabel name="Connectors"/> page.
+
+Ensure that data from the Apache Kafka topics is successfully transferred to the
+target IBM MQ queue.
+
+</TabItem>
```
Comment (ArthurFlag) [inline docs/products/kafka/kafka-connect/howto/ibm-mq-sink-connector.md:108]:
```suggestion
  </TabItem>
```

File: docs/products/kafka/kafka-connect/howto/ibm-mq-sink-connector.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+```bash
+avn service connector create SERVICE_NAME @ibm_mq_sink_connector.json
+```
+
+Parameters:
+
+- `SERVICE_NAME`: The name of your Aiven for Apache Kafka service.
+- `@ibm_mq_sink_connector.json`: This denotes the path to your JSON configuration file.
+
+</TabItem>
```
Comment (ArthurFlag) [inline docs/products/kafka/kafka-connect/howto/ibm-mq-sink-connector.md:123]:
```suggestion
  </TabItem>
```

File: docs/products/kafka/kafka-connect/howto/ibm-mq-sink-connector.md (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/kafka/kafka-connect/howto/ibm-mq-sink-connector.md]: Fixed

**Review by ArthurFlag — APPROVED** [review]

**Pull request merged**